Makefile

```diff
@@ -19,7 +19,7 @@ GOARCH := $(if $(GOARCH),$(GOARCH),"")
 GOENV  := GO15VENDOREXPERIMENT="1" CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH)
 CGOENV  := GO15VENDOREXPERIMENT="1" CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH)
 GO     := $(GOENV) go
-GOTEST := TEST_USE_EXISTING_CLUSTER=false go test
+GOTEST := TEST_USE_EXISTING_CLUSTER=false NO_PROXY=testhost go test
```

Comment:
Please add `testhost` to the end of `NO_PROXY` (split with `,`), but not override it